core/controllers/creator_dashboard.py

```diff
@@ -264,10 +284,23 @@ def post(self):
 class NewExplorationHandler(base.BaseHandler):
     """Creates a new exploration."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'title': {
+                'schema': {
+                    'type': 'unicode'
+                },
+                'default_value': feconf.DEFAULT_EXPLORATION_TITLE
+            }
+        }
+    }
+
     @acl_decorators.can_create_exploration
     def post(self):
         """Handles POST requests."""
-        title = self.payload.get('title', feconf.DEFAULT_EXPLORATION_TITLE)
+        title = self.normalized_payload.get(
+            'title', feconf.DEFAULT_EXPLORATION_TITLE)
```

Comment:
```suggestion
        title = self.normalized_payload.get('title')
```
There is no need for default value here since it is taken care by the schema handler

Reply:
Done.